src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -26,9 +27,19 @@ const ExecuteTxButton = ({
   const txNonce = isMultisigExecutionInfo(txSummary.executionInfo) ? txSummary.executionInfo.nonce : undefined
   const isPending = useIsPending(txSummary.id)
   const { setSelectedTxId } = useContext(ReplaceTxHoverContext)
+  const safeSDK = getSafeSDK()
```

Comment:
`getSafeSDK` will not react to changes in `ExternalStore` state. We should use the hook when accessing the value.